src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
+
+  const count = days * price;
+
+  if (days < 3) {
+    return count;
+  }
+
+  if (days < 7) {
+    return count - firstDiscount;
+  }
+
+  if (days >= 7) {
+    return count - secondDiscount;
+  }
```

Comment:
For better readability of your code you can use only 2 `if` statements here. There's no need to use `if (days < 3)` statement:
```js
  if (days >= 7) {
    return count - secondDiscount;
  }

  if (days >= 3) {
    return count - firstDiscount;
  }

return count;
```